components/Layout/Layout.js

```diff
@@ -1,11 +1,62 @@
 import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function Layout({ children }) {
-  return <MainContainer>{children}</MainContainer>;
+  const router = useRouter();
+
+  return (
+    <>
+      <MainContainer>{children}</MainContainer>
+      <nav>
+        <StyledList>
+          <ListItem $active={router.pathname === "/"}>
```

Comment:
very nice dynamic solution for the navigation highlighting ✨

Reply:
Thank you. :blush: